### PR TITLE
Missing GHO & Arbitrum USDC feeds

### PR DESCRIPTION
**Pull request by sakulstra**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

https://arbiscan.io/address/0xaf88d065e77c8cc2239327c5edb3a432268e5831
https://etherscan.io/address/0x40d16fc0246ad3160ccc09b8d0d3a2cd28ae6c2f

### CODE REVIEW

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -161,6 +161,7 @@ FROM
     ('gcr-global-coin-research', 'ethereum', 'GCR', 0x6307b25a665efc992ec1c1bc403c38f3ddd7c661, 4),
     ('gen-daostack', 'ethereum', 'GEN', 0x543ff227f64aa17ea132bf9886cab5db55dcaddf, 18),
     ('get-get-protocol', 'ethereum', 'GET', 0x8a854288a5976036a725879164ca3e91d30c6a1b, 18),
+    ('gho-stablecoin', 'ethereum', 'GHO', 0x40d16fc0246ad3160ccc09b8d0d3a2cd28ae6c2f, 18),
```
Comment (sakulstra) [inline models/prices/ethereum/prices_ethereum_tokens.sql:164]:
```suggestion
    ('gho-gho', 'ethereum', 'GHO', 0x40d16fc0246ad3160ccc09b8d0d3a2cd28ae6c2f, 18),
```

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 2]
```diff
@@ -84,6 +84,7 @@ FROM
     ('umami-umami-finance','arbitrum','UMAMI',0x1622bf67e6e5747b81866fe0b85178a93c7f86e3,9),
     ('uni-uniswap','arbitrum','UNI',0xfa7f8980b0f1e64a2062791cc3b0871572f1f7f0,18),
     ('usdc-usd-coin', 'arbitrum', 'USDC', 0xff970a61a04b1ca14834a43f5de4533ebddb5cc8, 6),
+    ('usdc-usd-coin-origin', 'arbitrum', 'USDC', 0xaf88d065e77c8cc2239327c5edb3a432268e5831, 6),
```
Comment (sakulstra) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:87]:
```suggestion
    ('usdc-usd-coin', 'arbitrum', 'USDC', 0xaf88d065e77c8cc2239327c5edb3a432268e5831, 6),
```

**Comment by sakulstra** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jeff-dude** [comment]
welcome to spellbook, @sakulstra!

we recently merged in the arbitrum usdc address, so i imagine the test will fail now after merging in main branch. we can resolve that and look to move forward with just `gho` addition

**Comment by sakulstra** [comment]
@jeff-dude sorry missed the message.

merged main and removed arbitrum main change.

**Review by jeff-dude — APPROVED** [review]
looks good, thanks!

**Pull request merged**